examples/blank/README.md

```diff
@@ -1,10 +1,10 @@
-# React-Static - Basic Example
+# React-Static - Blank Example
 
-This example is the most basic version of react-static available. It includes:
+This example is the blank version of react-static available. It includes:
 - Babel
 - CSS imports
 - Image imports
 - File imports
 - Automatic Routing
```

Comment:
I don't think `Automatic Routing` is part of the blank example, so maybe this could be removed as well?